components/LinkMainPage.vue

```diff
@@ -0,0 +1,26 @@
+<script setup>
+const props = defineProps({
+  cityName: String,
+});
+</script>
+
+<template>
+  <div class="container">
+    <router-link to="/">
```

Comment:
@PiotrDrechsler why not a [NuxtLink ](https://nuxt.com/docs/getting-started/routing#navigation) component?

Reply:
Fact. My mistake :) Fixed.